aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Orchestrator.java

```diff
@@ -292,7 +290,6 @@ private void stopObservingStorageChanges() {
 
     /**
      * Start syncing models to and from a remote API.
-     * @return A Completable that succeeds when API sync is enabled.
```

Comment:
is this to be deleted?

Reply:
I wonder why is this change showing up. I did not make this change and this change doesn't show up on android studio either. Moreover this method is void so its not valid anyways.